server/channels/web/context.go

```diff
@@ -235,6 +235,11 @@ func (c *Context) SetInvalidParamWithErr(parameter string, err error) {
 	c.Err = NewInvalidParamError(parameter).Wrap(err)
 }
 
+func (c *Context) HandlePayloadTooLargeError(err error) {
```

Comment:
Where exactly is this used? I don't see this referred anywhere.

Reply:
yeah, its unused. I've removed it now.